nemo/collections/common/parts/multi_layer_perceptron.py

```diff
@@ -51,11 +51,15 @@ def __init__(
     def last_linear_layer(self):
         return getattr(self, f'layer{self.layers - 1}')
 
-    def forward(self, hidden_states):
+    def forward(self, hidden_states, temperature: float | None = None):
```

Comment:
Is there another place you can add the temperature? I prefer MLP base classes to be as brain dead as possible. Even with an optional flag....